docs/src/build.md

```diff
@@ -28,11 +28,12 @@ cmake --build env/build
 
 ```bash
 source env/activate
-cmake -G Ninja -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
+cmake -G Ninja -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=clang-17 -DCMAKE_CXX_COMPILER=clang++-17
 cmake --build build
 ```
 
 > - To enable the ttnn/metal runtime add `-DTTMLIR_ENABLE_RUNTIME=ON`
+> - Clang 17 is the minimum required version when enabling the runtime.
```

Comment:
Should we update the table below? Around line 50.

Reply:
Yes! Thank you, updated